### PR TITLE
OOP school library: add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by MMGGYY66**
# OOP school library: add Person, Student and Teacher classes
- [Add .github/workflows and add inside it linters.yml and add Gemfile](https://github.com/MMGGYY66/school-library/commit/edc8d24356407b43b0fdeaf4927426d49b6e2936)
 
- [Add .rubocop.yml and run bundle install](https://github.com/MMGGYY66/school-library/commit/f4e590be4d2a4a15f980aa0722819eefae604e0e)
 
- [Create person.rb to contain the Person class](https://github.com/MMGGYY66/school-library/commit/798664f87f05ab70d64210ce49fdfe98e1f7ea31)
 
- [Create student.rb to contain the Student class](https://github.com/MMGGYY66/school-library/commit/060663a09cf9bbcef426939fb03c079cfcce9361)
 
- [Create teacher.rb to contain the Teacher class](https://github.com/MMGGYY66/school-library/commit/8588df5f4168598cd0bd84b02a0cdfead63b7e66)
 
- [Update README AND MIT files](https://github.com/MMGGYY66/school-library/commit/e80bc9eadadaf6193fccdc48f8a1badea01abd67)

- [Update README file](https://github.com/MMGGYY66/school-library/commit/cc3af826991ad7c5ff5900b96e353ce3646dde69)

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @parent_permission || is_of_age?
```
Comment (VICTOR-CHUKWU) [inline person.rb]: - [ ] the name of your private method is `of_age` not `is_of_age` kindly fix that

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @MMGGYY66  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (MMGGYY66) [reply person.rb]: Thank you.
Done

**Review by mukhammadrizooff — APPROVED** [review]
Hi @MMGGYY66, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good  implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**